PLUB/Configuration/CommonClass/InputTextView.swift

```diff
@@ -0,0 +1,227 @@
+//
+//  InputTextView.swift
+//  PLUB
+//
+//  Created by 김수빈 on 2023/01/08.
+//
+
+import SnapKit
+import UIKit
+import RxSwift
```

Comment:
내장 프레임워크를 먼저 임포트하고, 빈 줄로 구분하여 서드파티 프레임워크를 임포트해주실 수 있나요?

```suggestion
import UIKit

import RxSwift
import SnapKit
```

Reply:
저도 이 부분 수정해서 작성해야겠네요 !! 승현님 감사합니다 😁😁